webpack/assets/javascripts/react_app/common/MountingService.js

```diff
@@ -69,18 +55,6 @@ class ReactComponentElement extends HTMLElement {
     this._render();
   }
 
-  disconnectedCallback() {
-    try {
-      ReactDOM.unmountComponentAtNode(this.mountPoint);
```

Comment:
don't we need this unmount mechanism anymore?

Reply:
From reading the prs that added it, it was added for turbolinks, and we dont use that anymore